test/e2e/specs/test.js

```diff
@@ -7,6 +7,29 @@ module.exports = {
     .url('http://localhost:8080')
       .waitForElementVisible('#app', 5000)
       .assert.elementPresent('.randPersonBtn')
+      .pause(5000)
       .end()
+  },
+
+  'startship e2e tests': function (browser) {
+    browser
+        .url('http://localhost:8080')
+        .waitForElementVisible('#app', 5000)
+        //.click('.mdl-tabs__tab')
+        .setValue('.mdl-tabs__tab','Starship')
+        .click('.mdl-tabs__tab')
+        //.click('option[value="Starship"]') //selects the option but doesn't click
```

Comment:
Hi Alex,

The host is not define when the server is started, so by default it's
localhost. We can define a start host in the dev object in the config file.

The port can be retrieved from the following code.
var config = require('../../../config')
var port = process.env.PORT || config.dev.port

For question 2, the code will very likely break if the layout
changes(className or selection value change, etc.) It will be easy to fix
the problem to define a locator property file. If the layout change, just
need to change the property file instead of the code in the test case.

I'll update my code base and send another pull request in the next hour.

Thanks,
Jason

On Thu, May 19, 2016 at 9:46 AM, Alex Kramer <notifications@github.com>
wrote:

> In test/e2e/specs/test.js
> <https://github.com/alexkramer/force-vue/pull/23#discussion_r63880599>:
>
> >        .end()
> > +  },
> > +
> > +  'startship e2e tests': function (browser) {
> > +    browser
> > +        .url('http://localhost:8080')
> > +        .waitForElementVisible('#app', 5000)
> > +        //.click('.mdl-tabs__tab')
> > +        .setValue('.mdl-tabs__tab','Starship')
> > +        .click('.mdl-tabs__tab')
> > +        //.click('option[value="Starship"]') //selects the option but doesn't click
>
> Please remove commented code.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/alexkramer/force-vue/pull/23/files/867ffe20b8a64be2da42b05d5fecce30c1fd9865#r63880599>
>